pkgs/stdenv/linux/default.nix

```diff
@@ -332,7 +332,8 @@ in
         # and that can fail to load.  Therefore we upgrade `ld` to use newer libc;
         # apparently the interpreter needs to match libc, too.
         bintools = self.stdenvNoCC.mkDerivation {
-          inherit (prevStage.bintools.bintools) name;
+          pname = prevStage.bintools.bintools.pname + "-patchelfed";
+          inherit (prevStage.bintools.bintools) version;
```

Comment:
Looks good. 

`patchelfed` is a bit non-specific if I did not look at the implementation. When I read the description I initially though it's about `bootstrapTools` binutils. Maybe `-relinked-ld`?

Reply:
Good point, thanks.